packages/stk/stk_math/unit_tests/UnitTestStkMathMain.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2013, Sandia Corporation.
+// Under the terms of Contract DE-AC04-94AL85000 with Sandia Corporation,
```

Comment:
We may need to update the copyright notices with the new contract information at some point.